jit/src/main/java/org/dcsa/conformance/standards/jit/schema/SchemaParams.java

```diff
@@ -0,0 +1,49 @@
+package org.dcsa.conformance.standards.jit.schema;
+
+import io.swagger.v3.oas.models.media.Schema;
+import io.swagger.v3.oas.models.parameters.Parameter;
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
+
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+public class SchemaParams {
+
+  public static final String PORT_CALL_ID_REF = "#/components/parameters/portCallIDPathParam";
+  public static final String API_VERSION_MAJOR_REF = "#/components/parameters/Api-Version-Major";
+
+  public static Parameter getPortCallIDPathParam() {
+    return new Parameter()
+        .name("portCallID")
+        .in("path")
+        .required(true)
+        .description("The provider created identifier for the **Port Call**.")
+        .schema(getUUIDSchema());
+  }
```

Comment:
Personally I would see this as a public static method on a `PortCallID` class (which should also be used in the `PortCall` object. This way the definition for the PortCallID and the PathParam follow each other. In the YAML spec - this was always a problem for me to align the two...